task1/src/app/cource-item-date.directive.ts

```diff
@@ -0,0 +1,25 @@
+import {Directive, ElementRef, Input, OnInit} from '@angular/core';
+ 
+@Directive({
+    selector: '[fresh]'
+})
+export class CourceItemDateDirective{
+    @Input() courceDate = "";
+        
+    ngOnInit(){
+        const twoWeeksAgo= (date) => {
+            const twoWeeks = 14 * 24 * 1000 * 60 * 60;
+            const twoWeeksAgo = Date.now() - twoWeeks;
+        
+            return date > twoWeeksAgo;
+        }
+
+        if(twoWeeksAgo(new Date(this.courceDate)))
```

Comment:
this.courseDate - jest datą ponieważ jest z klasy CourceItem.ConductAt. nie trzeba go drugi raz zamieniac